src/main/java/de/unijena/cheminf/mortar/gui/views/SettingsView.java

```diff
@@ -312,6 +314,18 @@ protected void updateItem(IDisplayEnum item, boolean empty) {
                 default ->
                         throw new UnsupportedOperationException("Unknown property type " + tmpProperty.getName());
             }
+
+            Region backgroundRegion = new Region();
+            backgroundRegion.setPrefHeight(40);
+
+            // Set alternating colors
+            if (tmpRowIndex % 2 == 0) {
+                backgroundRegion.setStyle("-fx-background-color: #f5f5f5;");
+            } else {
+                backgroundRegion.setStyle("-fx-background-color: #ffffff;");
+            }
+            // Span across all columns
+            aGridPane.add(backgroundRegion, 0, tmpRowIndex, 2, 1);
```

Comment:
This whole setting of the background block should go to the beginning of the loop because right now, you are ignoring the first row (because the GridPane.add() calls above always contain tmpRowIndex++, so you start at index 1 here).

Reply:
... plus, you are styling a non-existing additional row at the end, right now.